packaging/osx/sharedframework/scripts/postinstall

```diff
@@ -6,8 +6,26 @@
 
 PACKAGE=$1
 INSTALL_DESTINATION=$2
+LIBCRYPTO_NAME=libcrypto.1.0.0.dylib
+LIBSSL_NAME=libssl.1.0.0.dylib
+
+if [ -z $OPENSSL_PATH ]; then OPENSSL_PATH=/usr/local/opt/openssl/lib; fi
+
+# Ugly, but we need this to actually get the version
+VERSION=${INSTALL_PKG_SESSION_ID##"com.microsoft.dotnet.sharedframework.Microsoft.NETCore.App."}
+VERSION=${VERSION%%".component.osx.x64"}
+CORE_LIB_FILE_PATH=$INSTALL_DESTINATION/shared/Microsoft.NETCore.App/$VERSION/System.Security.Cryptography.Native.dylib
+
 
 # A temporary fix for the permissions issue(s)
 chmod -R 755 $INSTALL_DESTINATION/shared
 
+if [ -e "$OPENSSL_PATH/$LIBSSL_NAME" ] && [ -e "$OPENSSL_PATH/$LIBCRYPTO_NAME" ]
+then
+    sudo install_name_tool \
```

Comment:
How does this work for tarball installation? Do we tell our customers to run this command themselves?